Kernel/Syscalls/uname.cpp

```diff
@@ -29,7 +29,7 @@ ErrorOr<FlatPtr> Process::sys$uname(Userspace<utsname*> user_buf)
 
     hostname().with_shared([&](auto const& name) {
         auto length = min(name->length(), UTSNAME_ENTRY_LEN - 1);
-        memcpy(buf.nodename, name->characters(), length);
+        AK::TypedTransfer<char>::copy(reinterpret_cast<char*>(buf.nodename), name->characters(), length);
```

Comment:
shouldnt `nodename` decay to a `char*`. so that a cast is not needed, or do the templates mess with that

Reply:
I tried, so no